src/components/FilterTop.js

```diff
@@ -164,6 +167,14 @@ function Filter(props) {
         <form onSubmit={handleSubmit}>
           <Grid container wrap="nowrap" direction="row">
             <Grid item className={classes.inputContainer}>
+              <TextField
```

Comment:
I'm not sure this field should go at the beginning of the filters.
Maybe after Capture ID would be better.

Reply:
@nmcharlton , I have updated as you requested